INSTALL.md

```diff
@@ -132,19 +110,19 @@ Clone the server configuration repository:
 ```shell
 mkdir /srv/OpenRailwayMap-setup
 git clone https://github.com/OpenRailwayMap/server-admin.git /srv/OpenRailwayMap-setup/
-cd /srv/OpenRailwayMap-setup/scripts
+cd /srv/OpenRailwayMap-setup/ansible/role/tileserver/files/scripts
```

Comment:
Thanks for updating this. Had me scratching my head last week. Eventually figured it out. Now we won't need to.